_pages/en/strategy-learning-automating-improving.md

```diff
@@ -54,68 +55,20 @@ IITB teams should:
 ##### Learn
```

Comment:
Should add:
Reach out to your [Security Champion network](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/SLF%20Forum%20Documents/Security%20Champions%20SLF-EN.pptx) to learn about security